composables/useProbeFilters.ts

```diff
@@ -12,12 +12,15 @@ export interface StatusOption {
 	outdatedOnly?: boolean;
 }
 
+export type SortOption = 'location' | 'tags' | 'name';
+
+export const SORTABLE_FIELDS: string[] = [ 'name', 'location', 'tags' ] as const;
```

Comment:
The type is not needed here, is it? I've seen this in a few other places too (ref definitions, etc).